sites/public/src/components/listings/search/ListingsSearchModal.module.scss

```diff
@@ -0,0 +1,13 @@
+.checkbox-group {
+  --inner-button-gap: var(--seeds-s3);
+  padding-top: var(--seeds-s4);
+  padding-bottom: var(--seeds-s4);
+}
+
+.checkbox-group > div > label {
+  color: var(--seeds-color-primary-dark) !important;
```

Comment:
We try to do everything we can to not use `important`, it can make things brittle / the order of CSS application become confusing. If this is the root class of the new component in seeds, if we're needing to override a style over here, we should make it a CSS variable over there :)